backend/src/test/java/com/redhat/cloud/notifications/routers/InternalServiceTest.java

```diff
@@ -320,7 +321,7 @@ private static void updateBundle(String bundleId, Bundle bundle, int expectedSta
                 .put("/internal/bundles/{bundleId}")
                 .then()
                 .statusCode(expectedStatusCode)
-                .contentType(familyOf(expectedStatusCode) == Family.SUCCESSFUL ? is(TEXT.toString()) : any(String.class));
+                .contentType(familyOf(expectedStatusCode) == Family.SUCCESSFUL ? containsString(TEXT.toString()) : any(String.class));
```

Comment:
did the text change? I mean is() is more precise than contains()

Reply:
Yes, Quarkus 1 returned the media type only, but Quarkus 2 also returns the UTF-8 encoding.